woocommerce/woocommerce
# Product Block Editor: improve a11y when creating a new Shipping class

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



This PR suggests some auto-focus of the `name` field by default when the modal appears.

### Changes proposed in this Pull Request:





Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new Product Editor
2. Create/edit a product
3. Go to the `Shipping` section
4. Select the `Add a new shipping class` option

<img width="450" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/29abe669-bac4-4f45-91c7-e30659436457">

5. Confirm the input auto-focuses

https://github.com/woocommerce/woocommerce/assets/77539/75718623-e707-4989-97c4-54e1a90f96b2



### Changelog entry







<details>

#### Significance




#### Type




#### Message 


</details>

<details>

#### Comment 


</details>
